docs/src/lang/user-operators.md

```diff
@@ -1,4 +1,4 @@
-# User-defined operators and recursive functions
```

Comment:
Again, we should not remove recursive operators and functions from the *language manual*. They are still a part of the language, even though we are not supporting them in the model checker.